src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -31,7 +31,11 @@ class ManageCLIResponse {
                 code: this.code,
             }
         };
-        if (this.list || this.status) json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        if (this.list || this.status) {
+            json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        } else {
+            json.response.message = typeof detail === 'string' ? detail : `${detail}`;
```

Comment:
can you add tests for this change

Reply:
From what I know if the type is `object` then `${detail}` will look like: [object Object]
For example nodejs REPL:

> > let aa = {a: 1}
> > console.log(aa)
> { a: 1 }
> > console.log(`${aa}`)
> [object Object]

If you add a small test for this change as @nadavMiz you can check it.

Reply:
@nadavMiz could you please explain what kind of tests you are suggesting here. Also, I have already modified some tests please check.

Reply:
I think the test you modified are good enough

Reply:
sure, thanks